skopt/tests/test_space.py

```diff
@@ -225,3 +227,38 @@ def test_space_api():
                       [(0.0, 1.0), (-5, 5), (0.0, 1.0), (0.0, 1.0), (0.0, 1.0),
                        (np.log10(1.0), np.log10(5.0)), (0.0, 1.0)]):
         assert_array_equal(b1, b2)
+
+
+def test_real_hypercube():
+    a = Real(2.0, 30.0, transform="hypercube")
+    for i in range(50):
+        yield (check_limits, a.rvs(random_state=i), 2, 30)
+
+    rng = np.random.RandomState(0)
+    X = rng.randn(100)
+    X_scaled = 28 * (X - X.min()) / (X.max() - X.min()) + 2
+    X_scaled = np.clip(X_scaled, 2.01, 30.01)
+
+    # Check transform
```

Comment:
Do we want a transform that does `0<= x <= 1` or `0 < x < 1`? The test here is for the latter but my guess would be that we want out transform to be the first?

Maybe instead of random values or in addition to them we should test explicitly the edge cases.